onnxruntime/contrib_ops/cpu/quantization/dynamic_quantize_matmul.cc

```diff
@@ -393,7 +407,7 @@ Status DynamicQuantizeMatMul::Compute(OpKernelContext* ctx) const {
 
     for (size_t gemm_idx = 0; gemm_idx < num_gemms; gemm_idx++) {
       auto& params = gemm_data_vec[gemm_idx];
-      params.A = reinterpret_cast<const float*>(a_data + helper.LeftOffsets()[gemm_idx]);
+      params.A = params.A = a_data + helper.LeftOffsets()[gemm_idx];
```

Comment:
I think `a_data` would need to be a `const float*` here. also, the `params.A = params.A =` assignment is redundant.